modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/display/context/RoleDisplayContext.java

```diff
@@ -189,9 +192,25 @@ public List<NavigationItem> getRoleAssignmentsNavigationItems(
 		String tabs2 = ParamUtil.getString(
 			_httpServletRequest, "tabs2", "users");
 
+		Map<String, String[]> parameterMap = portletURL.getParameterMap();
+
+		long roleId = GetterUtil.getLong(parameterMap.get("roleId")[0]);
+
+		Role role = RoleServiceUtil.fetchRole(roleId);
+
+		if (Objects.equals(RoleConstants.ADMINISTRATOR, role.getName())) {
+			_assigneeTypeNames = Arrays.stream(
```

Comment:
Avoid using stream. Try using TransformUtils please.

Reply:
I discovered that ArrayUtil also does the work 😁 
https://github.com/liferay-commerce/liferay-portal/pull/4499/commits/3e74aec33c9421e2e114a3fb50a3ce34a85c62b8#diff-0430dbaecaa63fc25fe61f0406791e480443aea0aad90681cfcd5aefc89a31b5R388